### PR TITLE
Set a production environment for production builds, fixes #244

### LINKED ISSUE #244: Use non-development build of React for non-development
We're spending most of our cycles on PropType checking

### PR DESCRIPTION
**Pull request by victorporof**
Signed-off-by: Victor Porof <vporof@mozilla.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,11 @@
...
-    "start": "node build/main.js --run",
-    "dev": "node build/main.js --run-dev",
-    "package": "node build/main.js --package",
-    "test": "nyc -s node build/main.js --test",
-    "lint": "nyc node build/main.js --lint",
+    "start": "NODE_ENV=\"production\" node build/main.js --run",
+    "dev": "NODE_ENV=\"development\" node build/main.js --run-dev",
+    "package": "NODE_ENV=\"production\" node build/main.js --package",
+    "test": "NODE_ENV=\"production\" nyc -s node build/main.js --test",
+    "lint": "NODE_ENV=\"production\" nyc -s node build/main.js --lint",
```
Comment (Mossop) [inline package.json]: Sadly for all involved this sort of environment variable setting doesn't work on Windows.

Comment (victorporof) [inline package.json]: Fudge.

Comment (victorporof) [inline package.json]: Remind me why we ditched webpack again? 😭

**Comment by victorporof** [comment]
What about this way @Mossop?

**Comment by Mossop** [comment]
It certainly runs, is there a way I can check it is doing what we expect?

**Comment by victorporof** [comment]
This should fix https://github.com/mozilla/tofino/issues/244 

So take a look and see if `npm run dev` shows some failed props vaildations in the console, while `npm start` doesn't.

**Comment by victorporof** [comment]
Related: https://github.com/mozilla/tofino/pull/526

**Comment by Mossop** [comment]
LGTM

**Pull request merged**